examples/regression/1-quickstart/plot_heteroscedastic_1d_data.py

```diff
@@ -0,0 +1,197 @@
+"""
+==========================================================
+Estimate the prediction intervals of 1D homoscedastic data
+==========================================================
+
+:class:`mapie.regression.MapieRegressor` is used to estimate
```

Comment:
Update the docstring to explain the introduction of heteroscedastic noise.

Reply:
done